src/cephadm/cephadm

```diff
@@ -13,6 +13,7 @@ import os
 import platform
 import pwd
 import random
+import requests
```

Comment:
We need to add python3-requests to the cephadm package dependency list as well

Reply:
@dsavineau does the latest commit adding the package dependencies look correct?

Reply:
this dependency might be a problem if it isn't part of the base python install...

Reply:
IIUC urllib.request is in the python standard library?  Can we use that instead?  https://docs.python.org/3/library/urllib.request.html#module-urllib.request